README.md

```diff
@@ -84,9 +88,13 @@ This list was inspired by the [awesome-broadcasting list](https://github.com/ebu
 ## Forums
 
 - [dcinemaforum](http://dcinemaforum.com/forum/index.php) – general digital cinema forum, with a slight focus on the DCP mastering software OpenDCP.
+- [Film-Tech Digital Cinema Forum](http://www.film-tech.com/vbb/forum/main-forum) - General forum, with a focus on projection
+- [Filmvorführerforum](http://www.filmvorfuehrer.de/) - (German) General cinema tech forum
 - [DCP-o-matic User Forum](https://dcpomatic.com/forum/) – user forum of the DCP mastering software DCP-o-matic.
+- [DCP-o-matic Mailing Lists](https://lists.dcpomatic.com/postorius/lists/) - Mailing lists for DCP-o-matic.
```

Comment:
Maybe the `DCP-o-matic` forum link is enough? The mailing list is essentially an archive by now.